kubectl/assets/nginx-deployment.yaml

```diff
@@ -14,6 +14,6 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: nginx:1.7.9
+        image: quay.io/bitnami/nginx:latest
```

Comment:
Is there a particular reason it's now using `latest`? Typically versions are locked to avoid trouble with using the latest.

Reply:
see my earlier explanation / answer to this same question.

Reply:
Yep, all good.